nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -36,11 +36,7 @@
     MegatronPretrainingSampler,
 )
 from nemo.collections.nlp.data.language_modeling.megatron.gpt_dataset import build_train_valid_test_datasets
-from nemo.collections.nlp.data.language_modeling.megatron.gpt_fim_dataset import (
-    GPTFIMDataset,
-    GPTFIMDatasetConfig,
-    is_dataset_built_on_rank,
```

Comment:
We reference this import:
https://github.com/NVIDIA/NeMo/blob/e4f4b6ed4a45eaccb48831a8d5c6e5f08267c02f/nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py#L1372